spec/feature/feature_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe 'Feature', type: :feature do
+  before(:each) do
+    visit ''
+  end
+  let(:user) { FactoryBot.create(:user) }
+  context 'open table of users by' do
+    it 'sign in', js: true do
+      expect(page).to have_current_path '/users/sign_in'
```

Comment:
Feature tests should not check path. They should check view content (https://m.patrikonrails.com/how-i-test-my-rails-applications-cf150e347a6b System tests).
Controller tests should check path.

Reply:
Use route path like `user_sign_in_path` instead hard code.